tests/functional/test_endpoints.py

```diff
@@ -32,6 +34,32 @@
     'lex-runtime': 'runtime.lex',
 }
 
+BLACKLIST = [
+    'mobileanalytics',
+]
+
+
+def test_endpoint_matches_service():
+    backwards_renames = dict((v, k) for k, v in SERVICE_RENAMES.items())
+    session = get_session()
+    loader = session.get_component('data_loader')
+    expected_services = set(loader.list_available_services('service-2'))
+
+    pdir = os.path.dirname
+    endpoints_path = os.path.join(pdir(pdir(pdir(__file__))),
```

Comment:
You could have loaded this with the ``JSONFileLoader.load_file()``: https://github.com/boto/botocore/blob/develop/botocore/loaders.py#L156